config/sidekiq.yml

```diff
@@ -12,6 +12,7 @@
 # even put in dynamic logic, like a host-specific queue.
 # http://www.mikeperham.com/2013/11/13/advanced-sidekiq-host-specific-queues/
 :queues:
+  - [async_database_migration, 1]
```

Comment:
We could bump the priorities for existing jobs until action mailbox_routing by +1

So that `async_migration` remains a lower priority compared to other jobs

Reply:
Made the changes